stockflux-securities-master/src/components/SecuritiesTable.css

```diff
@@ -0,0 +1,71 @@
+body {
+  background-color: var(--middle-background);
+  font-family: var(--font-roboto-sans-serif);
+  color: var(--text-grey-lighter);
+  user-select: none;
+  margin: 0;
+  border: 1px solid var(--dark-background);
+  overflow-y: hidden;
+}
+
+.securities-container {
+  overflow: hidden;
+  font-family: "Roboto", sans-serif;
```

Comment:
Should the font also be a CSS variable?

Reply:
Removed this as it's being imported with stock flux components.